e2e/app/fund.go

```diff
@@ -38,24 +37,24 @@ func accountsToFund(network netconf.ID) []common.Address {
 	case netconf.Staging:
 		return []common.Address{
 			fbDev,
-			contracts.StagingCreate3Deployer(),
-			contracts.StagingDeployer(),
-			contracts.StagingProxyAdminOwner(),
-			contracts.StagingPortalAdmin(),
-			contracts.StagingAVSAdmin(),
-			eoa.MustAddress(netconf.Staging, eoa.TypeRelayer),
-			eoa.MustAddress(netconf.Staging, eoa.TypeMonitor),
+			eoa.StagingCreate3Deployer(),
+			eoa.StagingDeployer(),
+			eoa.StagingProxyAdminOwner(),
+			eoa.StagingPortalAdmin(),
+			eoa.StagingAVSAdmin(),
+			eoa.MustAddress(netconf.Staging, eoa.RoleRelayer),
+			eoa.MustAddress(netconf.Staging, eoa.RoleMonitor),
 		}
```

Comment:
you currently have two ways to access an eoa address, lets align on only a single way, I suggest: `eoa.MustAddress(netconf.Staging, eoa.RoleRelayer)`